src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapOasis.java

```diff
@@ -259,4 +259,19 @@ public void vuapCheckSlotSizesOasis(Page page, String slot) {
 
     VuapAssertions.verifyVideoAdSize(vuap, videoAdHeight, adSlotHeight, MAX_AUTOPLAY_MOVIE_DURATION);
   }
+
+  @Test(
+      dataProviderClass = AdsDataProvider.class,
+      dataProvider = "adsVuapClickToPlayDesktop",
+      groups = {"AdsVuapClickToPlayDesktop"}
+  )
+  public void vuapClickToPlayShouldStartPlayingAdvertisementAfterClickOnPlayIcon(Page page, String slot) {
+    AdsBaseObject ads = openPageWithVideoInLocalStorage(page, VuapVideos.VIDEO_10s);
```

Comment:
Wouldn't it be easier/safer to prepare a test page with given video instead of forcing Porvata to display specific video? This feature works with Porvata only and it uses slightly different flow than regular VUAP